Cargo.toml

```diff
@@ -41,3 +41,34 @@ incremental = false
 [profile.release-dhat]
 inherits = "release"
 debug = 1
+
+# Dependencies used in more than one place are specified here in order to keep versions in sync:
+# https://doc.rust-lang.org/cargo/reference/workspaces.html#the-dependencies-table
+[workspace.dependencies]
+apollo-compiler = "=1.0.0-beta.15"
+apollo-parser = "0.7.6"
+apollo-smith = { version = "0.5.0", features = ["parser-impl"] }
+async-trait = "0.1.77"
+http = "0.2.11"
+once_cell = "1.19.0"
+reqwest = { version = "0.11.24", default-features = false, features = [
+    "rustls-tls",
+    "rustls-native-certs",
+    "gzip",
+    "json",
+    "stream",
+    "blocking",            # in router-fuzz
```

Comment:
could this feature be added only in router-fuzz?

Reply:
It turns out that yes. I’ve pushed another commit.